cpg_workflows/jobs/gcnv.py

```diff
@@ -592,7 +594,7 @@ def merge_calls(sg_vcfs: list[str], docker_image: str, job_attrs: dict[str, str]
         merge_job.command(f'bcftools merge {" ".join(batch_vcfs)} -Oz -o {merge_job.tmp_vcf} --threads 4 -m all -0')
 
     # now normlise the result, splitting multiallelics
-    merge_job.command(f'bcftools norm -m -any {merge_job.tmp_vcf} | bgzip -c > {merge_job.tmp_vcf_split}')
+    merge_job.command(f'bcftools norm -m -any -Oz -o {merge_job.tmp_vcf_split} {merge_job.tmp_vcf}')
```

Comment:
Do you actually want `-m -any` : if SNPs and indels should be merged into a single record, specify any
or `-m -both` : if both SNPs and indels should be merged separately into two records

Reply:
because this task is exclusively used for CNV data, I went with `any` as a catch all for splitting DUP/DEL/WT calls, wasn't really sure how these call types sit when the documentation explicitly notes SNVs and Indels :/